src/NuGetForUnity/Editor/NugetHelper.cs

```diff
@@ -72,6 +72,11 @@ public static class NugetHelper
         /// </summary>
         private static readonly Dictionary<string, NugetPackage> installedPackages = new Dictionary<string, NugetPackage>();
 
+        /// <summary>
+        ///     The dictionary of currently selected installed <see cref="NugetPackage" />s keyed off of their ID string.
+        /// </summary>
+        private static readonly Dictionary<string, NugetPackage> selectedInstalledPackages = new Dictionary<string, NugetPackage>();
```

Comment:
I think you can keep this is List instead of Dictionary. You are only using it by key in Unselect method but you are only calling that method when you determine the item among the Values of this dict so everything could be made to work with List just the same.